website/source/docs/guides/consul-aws.html.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: "docs"
+page_title: "Consul-AWS"
+sidebar_current: "docs-guides-consul-aws"
+description: |-
+  Consul-AWS provides a tool, which syncs Consul's and AWS CloudmMap's service catalog
+---
+
+# consul-aws
+
+[Consul-AWS](https://github.com/hashicorp/consul-aws/) syncs the services in an AWS CloudMap namespace to a Consul datacenter. Consul services will be created in AWS CloudMap and the other way around. This enables native service discovery across Consul and AWS CloudMap.
+This guide will describe how to configure and how to start the sync.
+
+## Authentication
+
+`consul-aws` need access to Consul and AWS for uni- and bidirectional sync.
+
+For Consul, the process accepts both the standard CLI flags, `-token` and the environment variables `CONSUL_HTTP_TOKEN`. This should be set to an Consul ACL token if ACLs are enabled.
+
+For AWS, `consul-aws` uses the default credential provider chain to find AWS credentials. The default provider chain looks for credentials in the following order:
+1. Environment variables.
+2. Shared credentials file.
+3. If your application is running on an Amazon EC2 instance, IAM role for Amazon EC2.
+
+## Configuration
+
+There are two subcommands available on `consul-aws`:
+
+* version: display version number
+* sync-catalog: start syncing the catalogs
+
+The version subcommand doesn’t do anything besides showing the version, so lets focus on sync-catalog. The following flags are available:
+
+* A set of parameters to connect to your Consul Cluster like `-http-addr`, `-token`, `-ca-file`, `-client-cert`, and everything else you might need in order to do that
+* `-aws-namespace-id`: The AWS namespace to sync with Consul services.
+* `-aws-service-prefix`: A prefix to prepend to all services written to AWS from Consul. If this is not set then services will have no prefix.
+* `-consul-service-prefix`: A prefix to prepend to all services written to Consul from AWS. If this is not set then services will have no prefix.
+* `-to-aws`: If true, Consul services will be synced to AWS (defaults to false).
+* `-to-consul`: If true, AWS services will be synced to Consul (defaults to false).
+* `-aws-pull-interval`: The interval between fetching from AWS CloudMap. Accepts a sequence of decimal numbers, each with optional fraction and a unit suffix, such as "300ms", "10s", "1.5m" (defaults to 30s).
+* `-aws-dns-ttl`: DNS TTL for services created in AWS CloudMap in seconds (defaults to 60).
+
+Independent of how you want to user `consul-aws` it needs to be able to connect to Consul and AWS. Apart from making sure you setup up authenticated access, `-aws-namespace-id` is mandatory.
```

Comment:
how you want to user `consul-aws` > how you want to **use** `consul-aws` ?
sure you setup up authenticated access > sure you setup authenticated access